### PR TITLE
Add to FlxFramesCollection an option to add multiple spritesheets

### PR DESCRIPTION
**Pull request by Eventretta**
As you may know to import a spritesheet you need to call FlxAtlasFrames.fromPacker() or fromSparrow().

But during the development of a game, having to import multiple spritesheets might be needed.

Example: The Friday Night Funkin Mod named VS Tricky.

@KadeDev had to code multiple frames implementation for the Hellclown Tricky character by making multiple FlxSprite and hiding/showing them when needed.

Now in the FlxFramesCollection class there is a function that takes as a parameter an Array with FlxFramesCollection and returns a concat of all of them.

I had also the necessity during the development of my game too so

### CODE REVIEW

File: flixel/graphics/frames/FlxFramesCollection.hx [hunk 1]
```diff
@@ -62,6 +62,20 @@ class FlxFramesCollection implements IFlxDestroyable
...
+	/**
+	 * Merges multiple `FlxFramesCollection` and returns the result of the concatenation.
+	 * @param	FrameList	An array that contains various instances of `FlxFramesCollection` (they must contain a spritesheet, see the API for how to initialize using a spritesheet)
+	**/
+	public static function mergeSheets(sheets:Array<FlxFramesCollection>):FlxFramesCollection
+	{
+		var tex:FlxFramesCollection = null;
+		for (i in 0...sheets.length)
+		{
+			tex = tex.frames.concat(i.frames);
```
Comment (Geokureli) [inline flixel/graphics/frames/FlxFramesCollection.hx:74]: have you tested this? wouldn't this throw a null object ref?

Comment (Eventretta) [reply flixel/graphics/frames/FlxFramesCollection.hx:74]: yea i tested
for me works fine
but prob ur right i might edit it

Comment (Eventretta) [reply flixel/graphics/frames/FlxFramesCollection.hx:74]: > have you tested this? wouldn't this throw a null object ref? tex is null

btw i removed that null thing + added some more stuff

Comment (Geokureli) [reply flixel/graphics/frames/FlxFramesCollection.hx:74]: text is still null, in fact I you just added a compile warning

Comment (Eventretta) [reply flixel/graphics/frames/FlxFramesCollection.hx:74]: -.- i suck at coding but tf am i supposed to make an instance from nowhere?? Xd

Comment (Eventretta) [reply flixel/graphics/frames/FlxFramesCollection.hx:74]: i might create a new instance using new()

Comment (Geokureli) [reply flixel/graphics/frames/FlxFramesCollection.hx:74]: That's one way to do this. more importantly, I just want you to test your changes before making a PR

Comment (Eventretta) [reply flixel/graphics/frames/FlxFramesCollection.hx:74]: oops sorry just i dont really do PR i just do code locally

btw bout the issue

im gonna do the idea i had

**Comment by Eventretta** [comment]
@Geokureli im gonna fix this then im gonna push. thx you for helpjng man :)

**Comment by Eventretta** [comment]
i have alot of problems so im gonna cancel this.

**Comment by Geokureli** [comment]
be sure not to commit .DS_Store files, too

**Comment by Eventretta** [comment]
idk. do they cause problems

**Comment by Eventretta** [comment]
?

**Comment by Geokureli** [comment]
https://en.wikipedia.org/wiki/.DS_Store
They are used by your OS, they add nothing to the repo except bloat

**Comment by Eventretta** [comment]
oh ok, thanks for letting me know :)

**Pull request closed without merging**